src/modules/app/__tests__/fetchData.spec.js

```diff
@@ -76,7 +76,7 @@ describe('fetchData', () => {
     const data = await fetchData(endpoint, dispatch, oldPayload, params)
     const mappingError = new MappingError(
       defaultName,
-      'invalid json response body at undefined reason: Unexpected token I in JSON at position 0'
+      'invalid json response body at  reason: Unexpected token I in JSON at position 0'
```

Comment:
why is this change? :D

Reply:
Dont know but this is some internal foo from the Error API.